scripts/download_data.sh

```diff
@@ -70,6 +70,23 @@ if [[ "$DO_OLD" ]]; then
     DO_MR='true'
 fi
 
+# check the md5, OSX compat
+function check_md5 {
+    filename=$1
+    if [[ -e $(which md5sum) ]]; then
+        md5sum -c "${filename}.md5"
+    elif [[ -e $(which md5) ]]; then
+        md5 -r "${filename}" > "${filename}.tmp.md5"
+        diff -q "${filename}.tmp.md5" "${filename}.md5"
+        retval=$?
+        rm "${filename}.tmp.md5"
+        return $retval
+    else
+        echo "Unable to check md5. Please install md5sum or md5"
+        exit 1
```

Comment:
```suggestion
        return 0
```
I would not exit 1. Otherwise people cannot download anything at all. This way, they might be lucky